doc/source/introduction.rst

```diff
@@ -4,15 +4,11 @@ Introduction
 Supported Data Format
 ---------------------
 
-Pygac reads AVHRR GAC (Global Area Coverage) and LAC (Local Area Coverage)
-level 1b data from NOAA, which is described in the `POD`_ (NOAA-14 and
-before) and `KLM`_ (NOAA-15 and following) user guides. The data can be
-obtained from `NOAA CLASS`_, where you can also find a comprehensive
-`introduction`_.
-
-.. note::
-    Pygac is currently not able to read files with the CLASS archive
-    header included.
```

Comment:
Is Pygac now able to read that?

Reply:
from the code, we seem to be able to deal with archive headers: https://github.com/pytroll/pygac/blob/c8f3eca74075455a40dfc2afccc6ea994eaade9f/pygac/klm_reader.py#L704-L710

Hence the removal here :)

Reply:
That's great :D Thanks for clarifying.